clients/cloud/ruby/README.md

```diff
@@ -0,0 +1,68 @@
+# Overview
+
+Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using the [ ZenDesk Ruby Client for Apache Kafka](https://github.com/zendesk/ruby-kafka).
+
+
+# Prerequisites
+
+* [Bundler](https://bundler.io/) installed on your machine. Install via `gem install bundler`
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
+* Local file with configuration parameters to connect to your Confluent Cloud instance ([how do I find those?](https://docs.confluent.io/current/cloud/using/config-client.html#librdkafka-based-c-clients)). Format the file as follows:
+    ```bash
+    $ cat ~/.ccloud/ccloud.config
```

Comment:
Should be librdkafka.config to avoid ambiguity with `~/.ccloud/config` (which is the ccloud tool config file)

Reply:
Fair enough. I opted for `ccloud.config` because of this client not being based on librdkafka. This is better for consistency across the examples as well

Reply:
Oh, right, librdkafka.config is indeed a bad name, but so is ccloud.config.  Maybe example.config ?